Healthcare.html

```diff
@@ -193,7 +193,66 @@
               <h1 style="font-weight: 300">Healthcare – Delivery</h1>
 			<p style="color: #FFFFFF; max-width: 90%; width: 800px" > Healthcare organisations are privy to an inundation of rich data that is enabling them to be more efficient. The challenge faced is how to utlise this data to deliver results and improvements. Too often information is gathered and stored in multiple sources and is not easily accessible. This lead to all sorts of inefficiencies such as time-consuming administration processes, repeated research, lack of patient history, and the list goes on. Optimal usage of this information will allow healthcare organisations to operate with greater insights and deliver better services.</p>
         </div>
-		<div align="center"> <img src="img/scroll-anim.gif" width="25px"> </div>
+		<div align="center">

```

Comment:
The align attribute in DIV is not supported in HTML5

Reply:
Changed and submitted new pull request.